tests/pw/pages/frontend/cart.page.ts

```diff
@@ -0,0 +1,27 @@
+import { BasePage } from '@pages/basePage';
+
+export default class CartPage extends BasePage {
+    quantityInputFieldFor(productTitle: string) {
+        return this.page.locator(`//div[@class="quantity"]/label[contains(text(), "${productTitle}")]/following-sibling::input`);
+    }
+
+    updateCartButton() {
+        return this.page.locator('//button[@name="update_cart"]');
+    }
+
+    quantityErrorElement() {
+        return this.page.locator('//td[@class="product-quantity"]/div[@class="required"]');
+    }
+
+    woocommerceErrorMessage() {
+        return this.page.locator('//div[@class="woocommerce-notices-wrapper"]/ul[@class="woocommerce-error"]/li');
+    }
+
+    async enterQuantityValue(productTitle: string, quantityValue: string) {
+        await this.quantityInputFieldFor(productTitle).fill(quantityValue);
+    }
+
+    async clickOnUpdateCartButton() {
+        await this.updateCartButton().click();
+    }
+}
```

Comment:
Locators are already defined in pages/selectors. Add new locators there if they are not already included. Using multiple files solely for locators can unnecessarily complicate the project.